src/execution/StepExecutionManager.ts

```diff
@@ -114,14 +111,12 @@ export class StepExecutionManager {
           return step.execution!
         }
 
-        if (response.txStatus === TransactionStatus.FAILED) {
-          // Figure out how to show rejections
-          console.log('transaction is rejected')
+        if (response.status === 'FAILED') {
           process = statusManager.updateProcess(step, process.type, 'FAILED', {
             error: {
-              message: 'Transaction failed',
-              htmlMessage: 'Transaction failed',
-              code: LifiErrorCode.TransactionFailed,
+              message: response.message,
+              htmlMessage: response.message,
+              code: LifiErrorCode.MultiSigTransactionFailed,
```

Comment:
Why do we need a custom error here?

Reply:
Imo, having separate context for error is better to avoid introducing logic in presentational components or functions returning plain text, such as `getProcessMessage` function.

To communicate with precise error message and details, went ahead with custom error message.